cache/in-memory/src/test.rs

```diff
@@ -181,7 +182,7 @@ pub fn cache_with_message_and_reactions() -> InMemoryCache {
 
 pub fn current_user(id: u64) -> CurrentUser {
     CurrentUser {
-        accent_color: Some(16711680),
+        accent_color: Some(16_711_680),
```

Comment:
hex code is much clearer
```suggestion
        accent_color: Some(0xff0000),
```